include/espi.h

```diff
@@ -0,0 +1,446 @@
+
+/*
+ * Copyright (c) 2019 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public APIs for eSPI driver
+ */
+
+#ifndef ZEPHYR_INCLUDE_ESPI_H_
+#define ZEPHYR_INCLUDE_ESPI_H_
+
+#include <misc/__assert.h>
+#include <misc/slist.h>
+
+#include <zephyr/types.h>
+#include <device.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief ESPI Driver APIs
+ * @defgroup espi_interface ESPI Driver APIs
+ * @ingroup io_interfaces
+ * @{
+ */
+
+/*
+ * eSPI VW channel packets contain 4 VWire states.
+ */
+#define ESPI_VW_SRC_ID0         0
```

Comment:
Since you are using `VWIRE_CHANNEL` below, it is better to do `ESPI_VWIRE_SRC_ID0` to be consistent.